getUserFeatureFlags/main.go

```diff
@@ -80,8 +80,7 @@ func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse,
 		return utils.ServerError(err)
 	}
 
-	origin := req.Headers["Origin"]
-	corsHeaders := middleware.GetCORSHeaders(origin)
+	corsHeaders := middleware.GetCORSHeaders(req.Headers)
```

Comment:
We need to update the version of feature flag dependency for this change post our deployment once.